examples/ffdebye/many_beads/input.replay.xml

```diff
@@ -0,0 +1,28 @@
+<simulation verbosity='medium'>
```

Comment:
Could you change the name of this folder to be more explanatory? E.g. change "many_beads" to "replay_with_many_beads"